data/vss-core/README.md

```diff
@@ -103,7 +103,7 @@ VERBOSE: SubscriptionHandler::publishForVSSPath: set value 8.927633333 for path
 VERBOSE: SubscriptionHandler::publishForVSSPath: set value 12.244000434875488 for path Vehicle.Speed
 ```
 
-The [VISS client](https://github.com/eclipse/kuksa.val/blob/master/kuksa_viss_client/README.md) can be used to verify that data actually is correctly interpreted.
+The [VISS client](https://github.com/eclipse/kuksa.val/blob/master/kuksa-client/README.md) can be used to verify that data actually is correctly interpreted.
```

Comment:
Do a file search for VISS, it shall only remain in those places where we refer to the VISS standard or the the VISS (websocket) protocol. (And in the latter cases we should possibly state that it is the websocket variant of VISS we support, not HTPPS/MQTT)

Reply:
Should I also touch the go client then as it also refers to VISS?

Reply:
I think it is best to ask @SebastianSchildt. Right now the Go-client only supports VISS (and not gRPC), so in one way it is correct to call it a VISS client. Maybe we should wait touching the internals of the Go client, until we have decided what to with it.

Reply:
OK. There is one occurrence of "GRPC" in the go client : https://github.com/eclipse/kuksa.val/blob/590198a35de7b2201bdd913750157bb9778a5214/kuksa_go_client/kuksa_viss/config.go but there is no actual implementation of a gRPC client indeed. I will leave it aside for now.